l10n-pocket/en/pocket/home.ftl

```diff
@@ -4,9 +4,16 @@
 
 ### URL: https:// dev.tekcopteg.com/
 
-pocket-home-get-right-to = Get right to the good stuff
+pocket-home-homepage-title = Home
+
+pocket-home-homepage-desc = When you find something you want to view later, put it in { -brand-name-pocket }.
```

Comment:
@maureenlholland @alexgibson as per [Maureen's comment](https://github.com/mozilla/bedrock/pull/11592#discussion_r870105506), I added the page description for the homepage here, however I noticed that this description is also the same one found in the `base.html` [template for Pocket](https://github.com/mozilla/bedrock/blob/main/bedrock/pocket/templates/pocket/base.html#L12). Would the one in the `base.html` file already show up if we don't have a description explicitly set for the homepage? I don't want to have multiples of the same description 👀

Reply:
Ideally every page should have a separate and unique description. If you override `{% block page_desc %}` in the home page template, then it will replace the default description with home page description. You won't get a duplicate.

Ideally we should localise what's in the default base template however. Right now it's hard coded in English, which isn't great. I'm not sure it makes sense to create a global `base.ftl` file just for this, so perhaps we should consider removing the default from the base template, and ensuring each page has its own unique title and description?

This could easily be spun out into a separate issue, if that makes sense.